src/parser/Parser.ts

```diff
@@ -1102,6 +1128,36 @@ export class Parser {
             return new Stmt.Return(tokens, toReturn);
         }
 
+        /**
+         * Parses a `goto` statement
+         * @returns an AST representation of an `goto` statement.
+         */
+        function labelStatement() {
+            let tokens = {
+                identifier: advance(),
+                colon: advance(),
+            };
+
+            consume("Labels must be declared on their own line", Lexeme.Newline, Lexeme.Eof);
```

Comment:
Since this is here, we might be able to drop the `checkNextNext()` function entirely!

Reply:
No, because this would throw errors for valid brightscript like
```brightscript
video = {
    url: "some_url"
}
```